Vic2ToHoI4/Source/HOI4World/HoI4Country.h

```diff
@@ -274,6 +276,7 @@ class Country
 	std::optional<int> capitalState;
 	std::optional<int> capitalProvince;
 	int oldCapital;
+	std::set<int> coreStates;
```

Comment:
Though it looks like the focus tree logic assumes all core states are present. Update either the world logic or the focus tree logic to be consistent.

Reply:
> Looks like these are unowned core states instead of all core states. Update the name appropriately.

> Though it looks like the focus tree logic assumes all core states are present. Update either the world logic or the focus tree logic to be consistent.

coreStates are _all_ states in the world that are considered to be a core part of the country. It seemed cleaner to have HoI4World determine all cores states of a country and store them in HoI4Country.h and then in HoI4FocusTree filter them to get only the ones owned by enemies.

Reply:
It looks like HoI4World was filtering by owner. You may wish to double-check that.

Reply:
> It looks like HoI4World was filtering by owner. You may wish to double-check that.

Ah, yes, my bad. I filtered in both places. I should remove the filter in HoI4World to make Country.h contain all cores, right? It's actually storing state IDs, so it's not that expensive.

Reply:
That would make the most sense.